wai-aria/role/roles.html

```diff
@@ -0,0 +1,137 @@
+<!doctype html>
+<html>
+<head>
+  <title>Simple Core ARIA Role Verification Tests</title>
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+  <script src="/resources/testdriver.js"></script>
+  <script src="/resources/testdriver-vendor.js"></script>
+  <script src="/resources/testdriver-actions.js"></script>
+  <script src="/wai-aria/scripts/aria-utils.js"></script>
+</head>
+<body>
+<script>
+
+/*
+
+Tests simple role assignment: <div role="alert">x</div>
+
+- Nested role structures (table>row>cell, etc) and
+  Abstract roles (e.g. widget, composite) are
+  commented with pointers to a separate test file.
+
+- ARIA extension specs (e.g. doc-*, graphics-*) are
+  commented with pointers to a separate spec directory.
+
+*/
+
+AriaUtils.assignAndVerifyRolesByRoleNames([
+  "alert",
+  "alertdialog",
+  "application",
+  "article",
+  // "associationlist" [AT-RISK: ARIA #1662] or possibly -> ./list-roles.html
+  // "associationlistitemkey" [AT-RISK: ARIA #1662] or possibly -> ./list-roles.html
+  // "associationlistitemvalue" [AT-RISK: ARIA #1662] or possibly -> ./list-roles.html
+  "banner",
+  "blockquote",
+  "button",
+  "caption",
+  // "cell" -> ./grid-roles.html
+  "checkbox",
+  "code",
+  // "columnheader" -> ./grid-roles.html
+  "combobox",
+  // "command" -> ./abstract-roles.html
+  // "comment" -> [AT-RISK: ARIA #1885]
+  "complementary",
+  // "composite" -> ./abstract-roles.html
+  "contentinfo",
+  "definition",
+  "deletion",
+  "dialog",
+  // "directory" -> FAIL. WONTFIX. Deprecated in ARIA 1.2; re-mapped to list role.
```

Comment:
Shouldn't this still be tested to check that computed role is `list`? (i.e. synonym-roles.html)

Reply:
Will add that to https://github.com/web-platform-tests/interop-2023-accessibility-testing/issues/10